agent/config/builder.go

```diff
@@ -18,6 +18,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hashicorp/consul/lib/stringslice"
```

Comment:
@kisunji was this in line with how we wanted to organize imports?

Reply:
No it should be in the bottom import group!

The catch with goimports and most other import formatting tools is that it preserves import groups by default. So this line will never be automatically moved to the bottom group because the format tools assume you whitespaced around it for a reason